pkg/storage/blockdev.go

```diff
@@ -190,3 +198,27 @@ func FilterEFISystemPartitions(devices []BlockDev) ([]BlockDev, error) {
 	}
 	return esps, nil
 }
+
+// GetMountpointByDevice gets the mountpoint by given
+// device name. Returns on first match
+func GetMountpointByDevice(devicePath string) (string, error) {
+	mountsPath, err := os.Open(LinuxMountsPath)
+	if err != nil {
+		return "", err
+	}
+
+	defer mountsPath.Close()
+	scanner := bufio.NewScanner(mountsPath)
+	scanner.Split(bufio.ScanLines)
+
+	var mountpoint string
+	for scanner.Scan() {
+		deviceInfo := strings.Split(scanner.Text(), LinuxMountsPathDelimiter)
```

Comment:
nit: use `strings.Fields` instead of `strings.Split`. It takes into account also consecutive spaces (not an issue with /proc/mounts but it's more robust in general)

Reply:
Done